server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/util/ReplacingServiceImpl.java

```diff
@@ -61,6 +64,7 @@
 
   private static final String ERROR_REPLACING = "Error replacing on text: %s with the pattern: %s";
   private static final int INDIVIDUAL_WORD_VALID_LENGTH = 322;
+  public static final String CONTINUATION_PREFIX = "      -    ";
```

Comment:
This may also go to the `PreprocessingConstants`

Reply:
updated